commands/review_apps/enable.js

```diff
@@ -0,0 +1,75 @@
+const cli = require('heroku-cli-util')
+const co = require('co')
+const api = require('../../lib/api')
+const KolkrabbiAPI = require('../../lib/kolkrabbi-api')
+
+module.exports = {
+  topic: 'reviewapps',
+  command: 'enable',
+  description: 'enable review apps and/or settings on an existing pipeline',
+  help: `Example:
+
+    $ heroku reviewapps:enable -p mypipeline --a myapp --autodeploy --autodestroy
+    Enabling review apps ...
+    Enabling auto deployment ...
+    Enabling auto destroy ...
+    Configuring pipeline... done`,
+  needsApp: false,
+  needsAuth: true,
+  wantsOrg: false,
+  args: [],
+  flags: [
+    {
+      name: 'pipeline',
```

Comment:
Here we could use `flags.pipeline` as well.

Reply:
Updated.